File: core/src/ten_runtime/binding/python/interface/ten_runtime/log_level.py
#
# Copyright © 2025 Agora
# This file is part of TEN Framework, an open source project.
# Licensed under the Apache License, Version 2.0, with certain conditions.
# Refer to the "LICENSE" file in the root directory for more information.
#
from enum import IntEnum


class LogLevel(IntEnum):
    INVALID = 0
    VERBOSE = 1
    DEBUG = 2
    INFO = 3
    WARN = 4
    ERROR = 5
    FATAL = 6

File: core/src/ten_runtime/binding/go/interface/ten_runtime/log_level.go
//
// Copyright © 2025 Agora
// This file is part of TEN Framework, an open source project.
// Licensed under the Apache License, Version 2.0, with certain conditions.
// Refer to the "LICENSE" file in the root directory for more information.
//

package ten

// LogLevel is the log level.
type LogLevel int32

const (
	// LogLevelVerbose is the verbose log level.
	LogLevelVerbose = 1

	// LogLevelDebug is the debug log level.
	LogLevelDebug = 2

	// LogLevelInfo is the info log level.
	LogLevelInfo = 3

	// LogLevelWarn is the warn log level.
	LogLevelWarn = 4

	// LogLevelError is the error log level.
	LogLevelError = 5

	// LogLevelFatal is the fatal log level.
	LogLevelFatal = 6
)
